Main.java

```diff
@@ -0,0 +1,12 @@
+//TIP To <b>Run</b> code, press <shortcut actionId="Run"/> or
+// click the <icon src="AllIcons.Actions.Execute"/> icon in the gutter.
+public class Main {
+    public static void main(String[] args) {
+Ball Ball1 = Lottery.getRandomBall();
```

Comment:
The variable name `Ball1` should follow camelCase naming conventions. Consider renaming it to `ball1`.